articles/machine-learning/resource-known-issues.md

```diff
@@ -425,6 +425,16 @@ method, or from the Experiment tab view in Azure Machine Learning studio client
   2. Enter `pip freeze` and look for `tensorflow`, if found, the version listed should be < 1.13
   3. If the listed version is a not a supported version, `pip uninstall tensorflow` in the command shell and enter y for confirmation.
 
+## Model explanations
+
+* **Sparse data not supported**: The model explanation dashboard breaks/slows down substantially with a large amount of features, therefore we currently do not support sparse data format. Additionally, general memory issues will arise with large datasets and large number of features. 
+
+* **Forecasting models not supported with model explanations**: Interpretability, best model explanation, is not available for Auto ML forecasting experiments that recommend the following algorithms as the best model: ForecastTCN, Average, Naive, Seasonal Average, and Seasonal Naive.
+
+* **Local explanation for data index**: The explanation dashboard does not support relating local importance values to a row identifier from the original dataset if the dataset is greater than 5000 datapoints as the dashboard randomly downsamples the data. However, the dashboard shows raw dataset feature values for each datapoint passed into the dashboard under the Individual feature importance tab. Users can map local importances back to the original dataset through matching the raw dataset feature values.
```

Comment:
I would probably clarify two things:
1. The "dataset" is the validation dataset in AutoML, not the training dataset (most customers will assume dataset means training dataset)
2. What happens when validation dataset size is less than 5000 samples? Answer: the `index` feature in AML studio will correspond to the index in the validation dataset.

(maybe we should also mention that for k-fold cross validation and Monte Carlo cross validation that even for size < 5000, the samples will be shuffled and the 1:1 correspondence will not hold, but this might be too much info for this one section)

Reply:
Great clarifications. Will add in, I think the validation question might be getting too technical :( I will leave it out but mention your two points.